Shared/Samples/Apply RGB renderer/ApplyRGBRendererView.swift

```diff
@@ -0,0 +1,70 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ApplyRGBRendererView: View {
+    /// An empty map.
+    @State private var map = Map()
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// The raster layer to apply RGB renderer.
+    @State private var rasterLayer: RasterLayer?
```

Comment:
```suggestion
    /// The raster layer to apply RGB renderer.
    private var rasterLayer: RasterLayer {
        map.basemap?.baseLayers[0] as! RasterLayer
    }
```

This can be a computed property.